recipes/wcurl/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "wcurl" %}
+{% set version = "2024.07.10" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/curl/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 962bb72e36e6f6cedbd21c8ca3af50e7dadd587a49d2482ab3226e76cf6dcc97
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - curl >=7.83.0
```

Comment:
Should Windows depend on `m2-bash`?

Reply:
If we used a selector for windows the package wouldn't be `noarch` anymore, right?

Reply:
Not necessarily. Please see [this doc]( https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies )

Reply:
Thanks!

Reply:
I couldn't find the documentation on how to execute the bash script on Windows. Is there any guidance somewhere? Right now, the script is being executed by `cmd.exe`, which obviously results in an error.

Reply:
Solved with a trampoline script